_posts/2024-03-06-february-2024-roundup.md

```diff
@@ -0,0 +1,106 @@
+An interesting project overview that decodes the encrypted protocol used by GoodWe smart meters and solar inverters to send metrics to the cloud, and then uses a Prometheus exporter to allow local monitoring of metrics. - François
+
+- [**Changing an ESP32 partition table over the air | by Florian Loitsch | The Toit Take**](https://blog.toit.io/changing-an-esp32-partition-table-over-the-air-276c86feeba8)<br>
+One solution for a very thorny problem with ESP-IDF: updating a partition table! - Eric
+
+- [**Doom running on Silicon Labs & Sparkfun Microcontrollers: A Quick Look**](https://community.silabs.com/s/share/a5UVm00000002Y5MAI/doom-running-on-silicon-labs-sparkfun-microcontrollers-a-quick-look?language=en_US&source=Email&detail=Newsletter&cid=eml-new-blu-022224&mkt_tok=NjM0LVNMVS0zNzkAAAGRcTWjktUpLJn8KTdKnqQPvKeWCtC_jHippIHL15RT0YNDIggH6jJYLXt44qTMDUsFieDGewcR4YbpO-WLcDfmY63FZBSRbMQZHMD9KZaFSJRCJTc)<br>
+Who doesn’t love a “we put Doom on an embedded device” post? - Eric
+
+- [**M0AGX / LB9MG - Abusing reserved interrupt vectors on Cortex-M for metadata**](https://m0agx.eu/cortex-m-reserved-vectors-as-metadata.html)<br>
+Follow this example of using a simple, small, single CRC32 checksum appended to a raw binary file for easy verification in bare-metal bootloaders, as opposed to the elaborate file formats of OS executables.
+
+- [**The State of Developer Ecosystem in 2023 Infographic | JetBrains**](https://www.jetbrains.com/lp/devecosystem-2023/embedded/)<br>
+JetBrains published their 2023 developer survey, and there's a way to filer it for embedded engineers. I love to use CLion, and I'm glad they are focusing more on embedded!
+
+- [**USB in a NutShell - Making sense of the USB standard**](https://www.beyondlogic.org/usbnutshell/usb1.shtml)<br>
+Oldie but a goodie; probably the best USB resource I’ve used that’s not digging into the standard. - Eric
+
+- [**Board Porting Guide — Zephyr Project Documentation**](https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#transition-to-the-current-hardware-model)<br>
+There’s a new Zephyr change for how users define their hardware that will be in v3.7 LTS, and this migration guide helps move to these new features. - Eric
+
+- [**Get up to speed with partial clone and shallow clone - The GitHub Blog**](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/)<br>
+A good explainer on partial and shallow clones, how they work, and when to use different forms. Great to help speed up downloading large vendor HALs. - Eric
+
+- [**What is Name Mangling in C++? – Abstract Expression**](https://abstractexpr.com/2023/01/03/what-is-name-mangling-in-cpp/)<br>
+Great explanation of name mangling (encoding a name and a signature to a symbol name) and how to demangle.
+
+- [**Making the most of your Shells History on Linux and macOS – Abstract Expression**](https://abstractexpr.com/2024/02/18/making-the-most-of-your-shells-history-on-linux-and-macos/)<br>
```

Comment:
Since this isn't directly tied to embedded, I think we should remove it

Reply:
@gminn the Shells History or Name Mangling one? Thank you so much for double checking all these!!

Reply:
Shells History